# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/*/gnome-libs14/db_185.patch
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

diff -Naur gnome-libs-1.4.17.orig/libgnome/gnome-dump.c gnome-libs-1.4.1.7/libgnome/gnome-dump.c
--- gnome-libs-1.4.1.7.orig/libgnome/gnome-dump.c	2002-01-07 00:12:05.000000000 +0100
+++ gnome-libs-1.4.1.7/libgnome/gnome-dump.c	2003-05-05 13:25:11.000000000 +0200
@@ -25,30 +25,7 @@
 #endif
 #include <sys/types.h>
 
-#ifdef PREFER_DB1
-#ifdef HAVE_DB1_DB_H
-# include <db1/db.h>
-#else
-# ifdef HAVE_DB_185_H
-#  include <db_185.h>
-# else
-#  include <db.h>
-# endif
-#endif
-
-#else
-
-#ifdef HAVE_DB_185_H
-# include <db_185.h>
-#else
-# ifdef HAVE_DB1_DB_H
-#  include <db1/db.h>
-# else
-#  include <db.h>
-# endif
-#endif
-
-#endif
+#include <db33/db_185.h>
 
 #include <string.h>
 #include <unistd.h>
diff -Naur gnome-libs-1.4.17.orig/libgnome/gnome-metadata.c gnome-libs-1.4.1.7/libgnome/gnome-metadata.c
--- gnome-libs-1.4.1.7.orig/libgnome/gnome-metadata.c	2002-01-07 00:12:05.000000000 +0100
+++ gnome-libs-1.4.1.7/libgnome/gnome-metadata.c	2003-05-05 13:05:40.000000000 +0200
@@ -52,27 +52,7 @@
 #include <ctype.h>
 #include <errno.h>
 
-#ifdef PREFER_DB1
-#ifdef HAVE_DB1_DB_H
-# include <db1/db.h>
-#else
-# ifdef HAVE_DB_185_H
-#  include <db_185.h>
-# else
-#  include <db.h>
-# endif
-#endif
-#else
-#ifdef HAVE_DB_185_H
-# include <db_185.h>
-#else
-# ifdef HAVE_DB1_DB_H
-#  include <db1/db.h>
-# else
-#  include <db.h>
-# endif
-#endif
-#endif
+#include <db33/db_185.h>
 
 #include "libgnomeP.h"
 #include "gnome-mime.h"
